modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -56,58 +56,60 @@ Below you can see what the core runtime’s hierarchy looks like:
-** `SOURCE_RESPONSE_SEND`: indicates an error ocurred in the source of the flow while sending a successful response.
-* `CIRTICAL`: indicates a severe error ocurred. These errors cannot be handled. 
-** `OVERLOAD`: indicates a problem of overloading ocurred and the execution was rejected. 
-*** `FLOW_BACK_PRESSURE`: indicates a problem of overloading ocurred at the source level. For example, using a HTTP listener as source.  
-** `FATAL_JVM_ERROR`: indicates that a fatal error ocurred, such as stack overflow. 
-
-== About Custom Error Types
-
-Custom error types can be defined when mapping errors or raising them. These errors
-require a specific custom namespace to distinguish them from other existing types
+** `TIMEOUT`: indicates timeout occurred while processing a message.
+** `UNKNOWN`: indicates an unknown or unexpected error occurred. This cannot be handled directly, only by handling `ANY`, to ensure backward compatibility in case more error types are added in future runtime versions.
+* `SOURCE`: indicates that an error occurred in the source of the flow.
+** `SOURCE_ERROR_RESPONSE_GENERATE`: indicates that an error occurred in the source of the flow generating the parameters of an error response. This error cannot be handled since the source has already executed the failing path.
+** `SOURCE_ERROR_RESPONSE_SEND`: indicates that an error occurred in the source of the flow sending an error response. This error cannot be handled since the source has already executed the failing path.
+* `SOURCE_RESPONSE`: indicates that an error occurred in the source of a flow while processing a successful response. These errors can only be propagated, since the source has already executed the successful path.
+** `SOURCE_RESPONSE_GENERATE`: indicates an error occurred in the source of the flow while generating the parameters of a successful response.
+** `SOURCE_RESPONSE_SEND`: indicates an error occurred in the source of the flow while sending a successful response.
+* `CRITICAL`: indicates a severe error occurred. These errors cannot be handled.
+** `OVERLOAD`: indicates a problem of overloading occurred and the execution was rejected.
+*** `FLOW_BACK_PRESSURE`: indicates a problem of overloading occurred at the source level. For example, using an HTTP listener as a source.
+** `FATAL_JVM_ERROR`: indicates that a fatal error occurred, such as stack overflow.
+
+== Custom Error Types
+
+To use custom error types, you need to define them either when mapping or when raising the errors.
+These errors require a specific custom namespace to distinguish them from other existing types
 within an application. This means that in an application using HTTP and DB, those
-two namespaces cannot be used. In fact, it's best to stay away from existing connector
+two namespaces cannot be used. It's highly recommended to stay away from existing connector
```

Comment:
> It's highly recommended to stay away from existing connector namespaces and instead define namespaces related to the particular Mule application name or context.

I'd re-order this sentence to give more importance that what the user **should** do. Something like:

> You should define namespaces related to your particular Mule application name or context, and avoid using existing connector namespaces.